plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -497,7 +510,7 @@ class CodeWhispererCodeScanManager(val project: Project) {
             val editorFactory = EditorFactory.getInstance()
             editorFactory.eventMulticaster.addDocumentListener(documentListener, project)
             editorFactory.addEditorFactoryListener(fileListener, project)
-            EditorFactory.getInstance().eventMulticaster.addEditorMouseMotionListener(
```

Comment:
This will get merge conflict as it was already merged.

Reply:
This branch is up to date so I don't thing thats the case.

Reply:
wait no....you can't remove this

Reply:
I did not remove the `eventMulticaster`. I just reused the variable which is declared above at L510.